website/content/tools/autoscaling/telemetry.mdx

```diff
@@ -141,6 +141,30 @@ policy handling.
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td>
+        <code>nomad-autoscaler.policy.total_num</code>
```

Comment:
nomad-autoscaler.policy.total_num is a duplicate of L170.

Reply:
<img width="1065" height="1180" alt="image" src="https://github.com/user-attachments/assets/e255578b-d8be-4c0a-9177-2034c88fb50e" />